test/functionalTest/cases/559_new_url_paths/ngsi10.test

```diff
@@ -721,13 +721,13 @@ Date: REGEX(.*)
 6.3. POST /ngsi10/contextEntities/EID
 ===================================
 HTTP/1.1 200 OK
-Content-Length: 556
+Content-Length: 557
 Content-Type: application/xml
 Date: REGEX(.*)
 
 <?xml version="1.0"?>
 <appendContextElementResponse>
-  <entityId type="" isPattern="false">
+  <entityId type="T" isPattern="false">
```

Comment:
I have a vague memory about some problem with type missing in conv ops... at that moment I think we though we have completely solved the issue, but it would be here again. Do you remember something like that?